CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
 # Changelog
 
+## v0.5.30
+
+*May 11 2022*
+
+This release includes support for Netint transcoding hardware, dynamic timeouts for Orchestrator discovery, protection against rounds with a zero block hash and a number of small Orchestrator bug fixes.
+
+### Breaking Changes 🚨🚨
+
+None
+
+### Features ⚒
+
+#### General
+- [#2348](https://github.com/livepeer/go-livepeer/pull/2348) Support Netint transcoding hardware (@cyberj0g)
+- [#2289](https://github.com/livepeer/go-livepeer/pull/2289) Add timeouts to ETH client (@leszko)
+- [#2282](https://github.com/livepeer/go-livepeer/pull/2282) Add checksums and gpg signature support with binary releases. (@hjpotter92)
+- [#2344](https://github.com/livepeer/go-livepeer/pull/2344) Use T.TempDir to create temporary test directory (@Juneezee)
+- [#2353](https://github.com/livepeer/go-livepeer/pull/2353) Codesign and notarize macOS binaries to be allowed to run without warnings on apple devices (@hjpotter92)
+- [#2351](https://github.com/livepeer/go-livepeer/pull/2351) Refactor livepeer.go to enable running Livepeer node from the code (@leszko)
+- [#2372](https://github.com/livepeer/go-livepeer/pull/2372) Upload test coverage reports to codecov (@hjpotter92)
+
+#### Broadcaster
+- [#2309](https://github.com/livepeer/go-livepeer/pull/2309) Add dynamic timeout for the orchestrator discovery (@leszko)
+- [#2337](https://github.com/livepeer/go-livepeer/pull/2337) Fix dynamic discovery timeout to not retry sending requests, but wait for the same request to complete (@leszko)
```

Comment:
I know we always have one bullet point per PR, but looking at it from the user perspective, the following 3 PRs provides one functionality: "Add dynamic timeout for the orchestrator discovery":
- #2309 => The actual PR introducing a feature
- #2337 => Fix for the PR above
- #2296 => Attempt before the above was implemented

It's not strictly related to this release, but I think we can consider creating a changelog which is more user facing and less PR-driven.

Saying that, we can keep it as it is for this release.

Reply:
Good point, I've kept the first one and removed the other two